### PR TITLE
feat(HACBS-1255): add rbac wildcard check to CI

### PR DESCRIPTION
**Pull request by srivickynesh**
JIRA : [HACBS-1255](https://issues.redhat.com/browse/HACBS-1255)
Signed-off-by: Sri Vignesh <sselvan@redhat.com>

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -101,6 +101,12 @@ jobs:
             git --no-pager diff ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen'
             exit 1
           fi
+      - name: Check wildcards in RBAC
+        run: |
+          if grep -q "*" /tmp/integration-service/config/rbac/role.yaml; then
```
[the remark this answers is not in the document]
Comment (srivickynesh) [reply .github/workflows/pr.yaml]: done

**Review by dirgim — APPROVED** [review]
LGTM

**Review by jsztuka — APPROVED** [review]
LGTM

**Pull request merged**